command_line/dials_import.py

```diff
@@ -571,9 +571,19 @@ def __call__(self, imageset_list):
                 if imageset.get_scan().is_still():
                     # make lots of experiments all pointing at one
                     # image set
-                    start, end = imageset.get_scan().get_array_range()
+
+                    # check if user has overridden the input - if yes, recall
+                    # that these are in people numbers (1...) and are inclusive
+                    if self.params.geometry.scan.image_range is not None:
+                        user_start, user_end = self.params.geometry.scan.image_range
+                        offset = imageset.get_scan().get_array_range()[0]
+                        start, end = ustart - 1, uend
```

Comment:
```suggestion
                        start, end = user_start - 1, user_end
```
Sorry, the above suggested change was incomplete (I should have modified it fully such that applying the suggested change would work).